src/main/resources/db/migration/V3__projections.sql

```diff
@@ -0,0 +1,17 @@
+create table reservation (
+    id bigserial not null,
+    canceled boolean,
+    user_id bigserial not null,
+    projection_id bigserial not null,
+    primary key (id)
+);
+
+alter table if exists reservation
+add constraint fk_user_id
+foreign key (user_id)
+references "user";
```

Comment:
You can inline FK declarations like this:

`user_id bigserial not null references "user"`

Reply:
Downside of this is that PSQL will auto-generate a constraint name, but it will be human-readable still